composer.json

```diff
@@ -15,6 +15,7 @@
     ],
     "require": {
         "php": "^5.3.6 || >=7.0 <7.2",
+        "doctrine/annotations": "^1.2",
```

Comment:
Can we put this as a [sugget](https://getcomposer.org/doc/04-schema.md#suggest)?

Those who will use these fixers probably already have it, those who don't will not install another scheme-tree invain